### PR TITLE
Lab 19, added changes to Instructions in Exercise 1 Task 2, fix for M…

### PR DESCRIPTION
**Pull request by preethidoss1221**
…aven build.

# Module: 19
## Lab:19 

# Lab 19, added changes to Instructions in Exercise 1 Task 2, Added instructions to fix Maven build by modifying pom.xml. The latest Maven release of 3.8.1 blocks HTTP artifactories, Students will have to use HTTPS instead of HTTP under PluginRepositories section of the pom.xml file.

### CODE REVIEW

**Comment by preethidoss1221** [comment]
Lab 19, added changes to Instructions in Exercise 1 Task 2, Added instructions to fix Maven build by modifying pom.xml. The latest Maven release of 3.8.1 blocks HTTP artifactories, Students will have to use HTTPS instead of HTTP under PluginRepositories section of the pom.xml file.

**Comment by FireyJoe** [comment]
I ran this lab and can confirm the problem with HTTP repositories and that the given solution works. Maven 3.8.1 release notes specify the change to disallow unsecure repositories: https://maven.apache.org/docs/3.8.1/release-notes.html. 
AZ-400 pull 84 comment 1

**Review by mkennten — CHANGES_REQUESTED** [review]
@preethidoss1221 Can you please run a quick spell/grammar check on our changes. Some examples: Identify (small i), url (all capitals), "Once the **Commit** button is enabled...", Vertical (small v), (please use the previous wording), "the" pom.xml file...

**Comment by preethidoss1221** [comment]
@mkennten  - Requesting for a review on commit id: c955de8 ,I have incorporated sentence correction. Thank you for your review.

File: Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md [hunk 1]
```diff
@@ -92,11 +92,19 @@ In this task, you will activate WhiteSource Bolt in the newly generated Azure De
 
 In this task, you will trigger a build within your Java code-based Azure DevOps project. You will use **WhiteSource Bolt** extension to identify vulnerable components present in this code.
 
-1.  On your lab computer, in the web browser window displaying the Azure DevOps portal with the **WhiteSource Bolt** project open, in the vertical menu bar on the left side, in the **Pipelines** section, click **Pipelines**.
-1.  On the **Pipelines** pane, select the **WhiteSourceBolt** build definition, click **Run pipeline** and, on the **Run pipeline** pane, click **Run** to trigger a build.
+1.  On your lab computer, in the web browser window displaying the Azure DevOps portal with the **WhiteSource Bolt** project open, in the vertical menu bar on the left side, click **Repos** and navigate to **WebGoat**.
+
+    > **Note**: As the project uses Maven to compile the project files, latest Maven release of 3.8.1 blocks HTTP artifactories. To successfully build the project, you will have to change the pom.xml file of the Project, to use HTTPS instead of HTTP. Perform below step for this purpose.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md:97]: "the" latest Maven release

Comment (mkennten) [reply Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md:97]: file of the Project ==> file of the project

Comment (mkennten) [reply Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md:97]: "Perform below steps" (plural)

File: Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md [hunk 2]
```diff
@@ -92,11 +92,19 @@ In this task, you will activate WhiteSource Bolt in the newly generated Azure De
...
-1.  On the **Pipelines** pane, select the **WhiteSourceBolt** build definition, click **Run pipeline** and, on the **Run pipeline** pane, click **Run** to trigger a build.
+1.  On your lab computer, in the web browser window displaying the Azure DevOps portal with the **WhiteSource Bolt** project open, in the vertical menu bar on the left side, click **Repos** and navigate to **WebGoat**.
+
+    > **Note**: As the project uses Maven to compile the project files, latest Maven release of 3.8.1 blocks HTTP artifactories. To successfully build the project, you will have to change the pom.xml file of the Project, to use HTTPS instead of HTTP. Perform below step for this purpose.
+
+1. Inside **webgoat-container** project file, identify the pom.xml file and click on **Edit** on the top right corner. Search for the section **pluginRepositories** inside the pom.xml file. Change the URL http://repository.apache.org/snapshots/ to https://repository.apache.org/snapshots/
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md:99]: "Inside the **webgoat-container** project file"

File: Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md [hunk 3]
```diff
@@ -92,11 +92,19 @@ In this task, you will activate WhiteSource Bolt in the newly generated Azure De
...
+1.  On your lab computer, in the web browser window displaying the Azure DevOps portal with the **WhiteSource Bolt** project open, in the vertical menu bar on the left side, click **Repos** and navigate to **WebGoat**.
+
+    > **Note**: As the project uses Maven to compile the project files, latest Maven release of 3.8.1 blocks HTTP artifactories. To successfully build the project, you will have to change the pom.xml file of the Project, to use HTTPS instead of HTTP. Perform below step for this purpose.
+
+1. Inside **webgoat-container** project file, identify the pom.xml file and click on **Edit** on the top right corner. Search for the section **pluginRepositories** inside the pom.xml file. Change the URL http://repository.apache.org/snapshots/ to https://repository.apache.org/snapshots/
+
+1. Once the **Commit** button is enabled after your edits, click on **Commit**.
+
+1. In the vertical menu bar on the left side, navigate to **Pipelines** section and click **Pipelines**. Observe the build, while it has automatically triggered due to the commit action performed in the previous step.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md:103]: "navigate to the **Pipelines** section"
"while it has  been automatically triggered by the commit performed in the previous step."

**Review by mkennten — CHANGES_REQUESTED** [review]
Hi, please correct the issues I commented on

File: Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md (hunk 3 above)
Comment (preethidoss1221) [reply Instructions/Labs/AZ400_M19_Implement_Security_and_Compliance_in_an_Azure_DevOps_pipeline.md]: Fixed additional sentence corrections at commit id: a541e0f . Thank you @mkennten for the review and guidance.

**Review by mkennten — APPROVED** [review]

**Pull request merged**